Sources/Ink/Internal/List.swift

```diff
@@ -59,26 +77,41 @@ internal struct List: Fragment {
                     reader.moveToIndex(fallbackIndex)
                     try addTextToLastItem()
                 }
-            case \.isNumber where list.isOrdered:
+            case \.isNumber:
+                guard case .ordered(_) = list.kind else {
```

Comment:
I recommend using a `switch` statements to handle enums, since that way we'll get a compiler error if we ever add another case, instead of silently failing. They're a bit more verbose, but I think the added safety is worth it.

Reply:
The reason why I felt this wasn't an issue was because we're looking for a specific case, not to handle all cases. So hypothetically a new case wouldn't be targeted here. Worst case the test suite would identify the regression.